### ISSUE TITLE
All equal sign(=) replaced with hyphen (-) in code block

### ISSUE THREAD

**Issue by abid-k** [giuspen/cherrytree#2209]
**Version, Operative system**
Windows 10, CT 0.99.53

**Describe the bug**
All of a sudden, all the equal signs (=) in all the code block in my notebook is replaced with hyphen (-)

**Screenshots**
![image](https://user-images.githubusercontent.com/36756466/215535421-9e2bbb19-1698-481d-a507-ac6e8a363294.png)

**Comment by abid-k** [comment]
More info:
1. No issues outside the code box.
2. Codeboxes in exported HTML also shows correctly.
3. Tried the new 0.99.54 version, issue still exists.
4. Tried with a new empty note book and issue is still there.
5. This change happened all of a sudden. Like 2 days back, things were fine and today, when I opened, everything is changed. There was no change in configuration or version that triggered this.
